resources/views/profile.blade.php

```diff
@@ -6,6 +6,7 @@
 
     <div class="col-md-8" >
         <div class="">
+            <a class="btn btn-success" href="{{route('Question::create')}}">Add Question</a>
             @forelse($levels as $level)
```

Comment:
I will insure that it will be visible to admin only
and I will add a middleware to prevent any access to controllers except admin